build.gradle

```diff
@@ -38,7 +38,7 @@ def grpc_google_common_protos = "0.1.9"
 def hamcrest_version = "1.3"
 def hadoop_version = "2.7.3"
 def jackson_version = "2.8.9"
-def spark_version = "1.6.3"
+def spark_version = "2.2.0"
```

Comment:
Consider using 2.2.1 (bugfix)?

Reply:
Yes, but 2.2.1 is on vote (not yet on Central). So I will update as soon as it will be available.

Reply:
Vote passed, just waiting the artifacts on Central  to update the PR.